backend/src/test/java/com/pickpick/message/application/ReminderServiceTest.java

```diff
@@ -61,293 +67,361 @@ class ReminderServiceTest {
+                List<ReminderResponse> foundReminders = reminderService.find(request, bom.getId()).getReminders();
+                List<Long> expectedIds = extractFutureFromTargetOrderedByRemindDateIds(bomReminders, target, count);
+
+                assertThat(foundReminders).extracting("id").containsExactlyElementsOf(expectedIds);
+            }
+        }
+
+        @DisplayName("messageId와 memberId로 단건 조회한다")
+        @Test
+        void findOne() {
+            Reminder target = bomReminders.get(5);
+            ReminderResponse response = reminderService.findOne(target.getMessage().getId(),
+                    target.getMember().getId());
+
+            assertThat(target.getId()).isEqualTo(response.getId());
+        }
+
+        @DisplayName("messageId와 memberId가 일치하는 리마인더가 없다면 예외가 발생한다")
+        @Test
+        void findOneThrowsException() {
+            Member kkojae = members.save(KKOJAE.create());
+            Message target = noticeMessages.get(0);
+
+            assertThatThrownBy(() -> reminderService.findOne(target.getId(), kkojae.getId()))
+                    .isInstanceOf(ReminderNotFoundException.class);
+        }
+
+        private List<Message> createAndSaveMessages(final Channel channel, final Member member) {
+            List<Message> messagesInChannel = Arrays.stream(MessageFixtures.values())
+                    .map(messageFixture -> messageFixture.create(channel, member))
+                    .collect(Collectors.toList());
+
+            for (Message fixture : messagesInChannel) {
+                messages.save(fixture);
+            }
+
+            return messagesInChannel;
+        }
+
+        private List<Reminder> saveRemindersAtDifferentTimes(final Member member, final List<Message> messages) {
+            List<Reminder> savedReminders = new ArrayList<>();
+            int totalSize = messages.size();
+
+            for (int i = 0; i < totalSize; i++) {
+                LocalDateTime remindDateTime = LocalDateTime.now().plusHours(i + 1);
+                Reminder reminder = reminders.save(
+                        new Reminder(member, messages.get(totalSize - i - 1), remindDateTime));
+                savedReminders.add(reminder);
+            }
+
+            return savedReminders;
+        }
+
+        private Message saveDummyMessage(final Member member, final Channel channel) {
+            Message message = new Message(UUID.randomUUID().toString(), "리마인더 설정 용 메시지", member, channel,
+                    LocalDateTime.now(), LocalDateTime.now());
+            return messages.save(message);
+        }
+
+        private List<Long> extractIds(final List<Reminder> reminders) {
+            return reminders.stream()
+                    .map(Reminder::getId)
+                    .collect(Collectors.toList());
+        }
+
+        private List<Long> extractOrderedByRemindDateIds(final List<Reminder> reminders) {
+            return reminders.stream()
```

Comment:
```suggestion
        private List<Long> extractIdsOrderedByRemindDate(final List<Reminder> reminders) {
            return reminders.stream()
```
이름 이렇게는 어떨까요? 추출하는 대상인 Ids가 extract 바로 뒤에 붙는게 가독성이 더 좋은 것 같아서요.

(리뷰 작성하고 보니 jpa 형식이라 친숙해보여서 그런가 싶네요)

Reply:
반영했습니다 😁